keyboards/georgi/georgi.c

```diff
@@ -0,0 +1,68 @@
+#include QMK_KEYBOARD_H
+
+bool i2c_initialized = 0;
+i2c_status_t mcp23018_status = 0x20;
+
+void matrix_init_kb(void) {
+  	steno_set_mode(STENO_MODE_GEMINI); // or STENO_MODE_BOLT
+
+    // (tied to Vcc for hardware convenience)
+    //DDRB  &= ~(1<<4);  // set B(4) as input
+    //PORTB &= ~(1<<4);  // set B(4) internal pull-up disabled
+
+    // unused pins - C7, D4, D5, D7, E6
+    // set as input with internal pull-up enabled
+    DDRC  &= ~(1<<7);
```

Comment:
Could all of these be converted to the GPIO Control functions? 
https://docs.qmk.fm/#/internals_gpio_control

It means that it will be a bit more verbose, but ...

Reply:
Maybe at some point in the future, somehow this feels pretty clear